src/myrtlespeech/model/deep_speech_2.py

```diff
@@ -2,6 +2,7 @@
 from typing import Tuple
 
 import torch
+from myrtlespeech.model.rnn import RNNState
```

Comment:
The import style used means that there should be a line break between these two:

```python
import python_package
from python_package import X

import installed_package
from installed_package import X

import myrtlespeech.X
from myrtlespeech import X
```

Reply:
nice done!

Reply:
The pre-commit hooks revert this to the original?

Reply:
If the hooks changing it then it is at least consistently wrong so we'll go with it for now!